util/options.cc

```diff
@@ -517,7 +517,8 @@ font_options_t::add_options (option_parser_t *parser)
     {"face-index",	0, 0, G_OPTION_ARG_INT,		&this->face_index,		"Set face index (default: 0)",		"index"},
     {"font-size",	0, default_font_size ? 0 : G_OPTION_FLAG_HIDDEN,
 			      G_OPTION_ARG_CALLBACK,	(gpointer) &parse_font_size,	font_size_text,				"1/2 numbers or 'upem'"},
-    /* TODO Add font-ppem / font-ptem. */
+    {"font-ppem",	0, 0, G_OPTION_ARG_INT,		&this->ppem,			"Set pixels per EM",			"0"},
```

Comment:
Looking at other options, looks like last argument is NOT default. It's the metavariable. Just see what the font-size and face-index have. Ie. the default goes into the argument help text. Eg. `"Set font point-size (default: 0; disabled)"`